css-typed-om/Overview.bs

```diff
@@ -784,10 +782,20 @@ The following are the arithmetic operations you can perform on dimensions:
         is set to |this|.
 </div>
 
-<div class=issue>
-    Define {{CSSNumericValue/equals()}}.
+<div algorithm="CSSNumericValue.equals(...|value|)">
+    The <dfn for=CSSNumericValue method>equals(...|value|)</dfn> method,
+    when called on a {{CSSNumericValue}} |this|,
+    must perform the following steps:
+
+    1. If the |value| item is the same type as the {{CSSNumericValue}} the method is being called on continue to step 2, else return false.
+
+    2. If the |value| item has the same type as the {{CSSNumericValue}} continue to the next step, else return false.
```

Comment:
* Same concerns as about the structure; reverse the conditional and just return false.
* I think you mean "value" here? Refer to `|this|’s {{CSSUnitValue/value}} internal slot` then (and same for |value|).